designer/server/src/test/scala/pl/touk/nussknacker/ui/api/OpenAPIExamplesValidator.scala

```diff
@@ -0,0 +1,109 @@
+package pl.touk.nussknacker.ui.api
+
+import com.networknt.schema.{InputFormat, JsonSchemaFactory, SpecVersion}
+import io.circe.{ACursor, Json}
+import io.circe.yaml.{parser => YamlParser}
+import org.scalactic.anyvals.NonEmptyList
+import pl.touk.nussknacker.engine.util.Implicits.RichScalaMap
+
+import scala.jdk.CollectionConverters._
+
+object OpenAPIExamplesValidator {
+
+  def validateExamples(specYaml: String): List[InvalidExample] = {
+    val specJson = YamlParser.parse(specYaml).toOption.get
+    val componentsSchemas =
```

Comment:
For me using Circe cursors is breaknecked. And at the end, it's not readable. 
I'm not insisting, but maybe you'd like to consider using `ujson` instead. It'd look like this:
```scala
    val specJson2 = YamlParser.parse(specYaml).toOption.get.noSpaces
    val componentsSchemas2 =
      ujson.read(specJson2)
        .obj("components")
        .obj("schemas")
```

Reply:
Looks better but 1) I don't want invest too much time for that and it would introduce some retesting 2) we would have to libs for operation on json, I don't know if it is worth it

Reply:
in the case of Circe we still operate on JSON. But it's ok. We can stay with this approach.

Reply:
BTW, take a look that I use optics like `downField(...).withFocus(...).top `. Is it possible in `ujson`?

Reply:
if you are asking if you can modify the ujson Value, the answer is yes.

Reply:
In my projects, I prefer ujson in tests because its readability is great. But in general, I use two features: extracting data from JSON and creating JSON from a String.

BTW. Underneath it's a mutable data structure, so I don't rather use it in production code.